crates/turbo-tasks-memory/src/task.rs

```diff
@@ -1275,20 +1291,92 @@ impl Task {
         }
     }
 
-    /// Access to a cell.
-    pub(crate) fn with_cell_mut<T>(
+    /// Read a cell.
+    pub(crate) fn read_cell(
```

Comment:
Bikeshedding but I prefer with_cell and with_cell_mut even it it not supposed to return anything